cmake/projects/aws-c-common/hunter.cmake

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2016-2018, Ruslan Baratov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    aws-c-common
+    VERSION
+    0.2.1
+    URL
+    "https://github.com/insufficientchocolate/aws-c-common/archive/hunter-0.2.1.tar.gz"
```

Comment:
> Please fork my repository or the original repository, in case I delete my fork or account

Please use https://github.com/hunter-packages/aws-c-common here

Reply:
Ok, should be fixed.